src/plugins/__init__.py

```
@@ -29,7 +29,8 @@ def auto_discover(self, module_path: str = "src.plugins") -> None:
         plugin_modules = [
             'openaq',
             'purpleair',
-            'waqi'
+            'waqi',
+            'jartic'
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)